AsomamecoApi/test/Controllers/RolesControllerTests.cs

```diff
@@ -0,0 +1,265 @@
+﻿using AutoFixture;
+using FluentAssertions;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Moq;
+using src.Controllers;
+using src.Models;
+using src.Services;
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace test.Controllers
+{
+    public class RolesControllerTests
+    {
+        private readonly IFixture _fixture;
+        private readonly Mock<IRoleService> _serviceMock;
+        private readonly RoleController _controller;
+
+        public RolesControllerTests()
+        {
+            _fixture = new Fixture();
+            _serviceMock = _fixture.Freeze<Mock<IRoleService>>();
+            _controller = new RoleController(_serviceMock.Object);
+        }
+
+        // Test Get All
+        [Fact]
+        public async Task GetAll_ShouldReturnOkResponse_WhenDataFound()
```

Comment:
Este test falla: Message: 
  Expected result to be assignable to Microsoft.AspNetCore.Mvc.ActionResult`1[System.Collections.Generic.IEnumerable`1[src.Models.Event]], but Microsoft.AspNetCore.Mvc.ActionResult`1[System.Collections.Generic.IEnumerable`1[src.Models.Role]] is not.